utils/releaser/src/ReleaseWorker/Release/MergeReleaseCandidateBranchReleaseWorker.php

```diff
@@ -34,7 +34,7 @@ public function getPriority(): int
     public function work(Version $version): void
     {
         $this->symfonyStyle->note('You need to create a merge commit, see https://github.com/shopsys/shopsys/blob/master/docs/contributing/merging-on-github.md for detailed instructions.');
-        $this->confirm('Confirm the release branch was merged to master');
+        $this->confirm(sprintf('Confirm "%s" branch was merged to "%s"', $this->createBranchName($version), $this->initialBranchName));
```

Comment:
It's problem with this whole stage (release). 
I should run the release stage on `base-branch`? If so, the `CheckChangelogForTodaysDateReleaseWorker` will not behave as expected (unable to find changelog entry as it was not yet merged and push to wrong branch). Running release stage on rc- branch results in this instruction being wrong (`[Manually] Merge "rc-v7-5-0" branch into "rc-v7-5-0"`)

Reply:
well, the `release` stage is designed this way:
It instructs you to run it on the proper branch, so the `$initialBranchName` is set properly, and then it checkouts to the `rc-xxx-branch` before the changelog is checked, see 3f43513c20ef55fd204f9da8e82ba863410c31d3

Reply:
I saw the checkout, but then it does not come together in my head :)
Can you please extract the checkout into separate release worker? It will be much clearer and we can add another worker if needed.

Reply:
makes sense. will do it

Reply:
fixed